packages/lesswrong/components/contents/SideItemLine.tsx

```diff
@@ -8,12 +8,14 @@ const styles = (theme: ThemeType) => ({
     [theme.breakpoints.down('xs')]: {
       display: "block",
     },
+    width: 13,
```

Comment:
Fills a gap in the hoverable area between the 4-px wide visual bit at the right edge, and the leftmost pixel of the side-items column in the page layout.